m4_sessions/1901-inning/Imposters.md

```diff
@@ -18,17 +18,21 @@ have some visibility on your own struggle and how to look at it from an outside
 
 Loryn Mason, Kylie Stanley, Mike Duke, Michael Gatewood:
 
-Introductions : 10 mins with full group - Name, mod, program, ice-breaker question. Acknowledge trauma, consent, and give the option to set own level of participation. 
+Introductions : (Mike) 10 mins with full group - Name, mod, program, ice-breaker question. Acknowledge trauma, consent, and give the option to set own level of participation. (Rock, Paper, Scissors). 
 
-Personal Experiences: Write on a note card: something that triggers your imposter syndrome. try statements like "'something happens' and I don't feel like I belong"
+Personal Experiences: (5 to 10 minutes) (Mike)
+ - Give short definition of imposter syndrome if needed.
+ - Write on a note card: something that triggers your imposter syndrome. try statements like "'something happens' and I don't feel like I belong". Give our own statements as examples. 
 
-Information about imposter syndrome: Teaching about imposter cycle/5 types of imposter syndrome from slides:
+Information about imposter syndrome: (5 to 10 minutes) (Michael) Teaching about imposter cycle/5 types of imposter syndrome from slides:
 https://docs.google.com/presentation/d/1Al72A0SQjA-3oLkSDIEDMdfAGpP0dSoAPZRXMu1nnVY/edit#slide=id.p
 
-Telling our stories: Hand out note cards randomly to each participant to read someone else's experience. Try and identify the type of IS is detailed (more than one is okay)
+Take a pom: 5 to 10 minutes
 
-What can we do about it: Go back to the slides and talk about strategies to combat each type of IS. Need to research more specific strategies than the ones on the slides. 
+Telling our stories: (15 minutes) (Kylie) Hand out note cards randomly to each participant to read someone else's experience. Try and identify the type of IS is detailed (more than one is okay). Emphasize self-empathy and helping others. 
```

Comment:
Maybe also set some norms for people on sharing in addition to the empathy you are pushing for